src/book/Book.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export default function BookList(props) {
+   return  <div key={props.book.id} className="book">
+        <h3>{props.book.title}</h3>
+        <div>
+            prix : {props.book.price / 100}€
+            <br/>Sujets :
+            <ul>
+                {props.book.subjects.map(
+                    subject => (<li key={subject.id}>{subject.name}</li>)
+                )}
+            </ul>
+            <br/>Niveau{props.book.levels.length>1?'x':''} : {props.book.levels.map(l => l.name).join(', ')}
```

Comment:
Nice pluralization 👍 .

In practice, using an i18n library makes pluralization easier. Even between French and English, pluralization is hard, but in Arabic there are 6 forms of pluralization… https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals

Reply:
👍